components/layout/blog-post/blog-post.hbs

```diff
@@ -0,0 +1,244 @@
+<div class="section--bg-prim-blue">
+
+    <div class="blog-post-hero section__inner">
+        <h1 class="blog-post-hero__title">If your team isn’t learning,
+            they’re falling behind. Fix it.</h1>
+        <div class="blog-post-hero__blog-author">
+            <div class="blog-post-hero__blog-author-img-container">
+                <img class="blog-post-hero__blog-author-img-container__author-image"
+                    src="http://placekitten.com/300/300" />
+            </div>
+            <div class="blog-post-hero__blog-author-info">
+                <span class="blog-post-hero__blog-author-info-name blog-author-name">By <a href="#">Meow
+                        Murray</a></span>
+                <p class="blog-post-hero__blog-author-info-title">President, Community & Customer Engagement</p>
+            </div><!-- .blog-post-hero__blog-author-info -->
+        </div><!-- .blog-post-hero__blog-author -->
+
+        <div class="blog-post-hero__time-social">
+            <div class="blog-post-hero__time-social-read-time">
+                <p class="blog-post-hero__time-social-read-time--p-text">Time</p>
+                <span class="blog-post-hero__time-social-read-time--p-time">10 minutes</span>
```

Comment:
I haven't gotten to the CSS here yet, but using `--p-text` and `--p-time` implies these are modifiers of the `__time-social-read-time` which doesn't really feel right. I might propose here a slightly different solution. Why not do this as one single element with no children, and use a ::before psuedo-element to apply the "Time" label?

Reply:
Updated.